doc/manual/acknowledgements.texinfo

```diff
@@ -0,0 +1,15 @@
+The PDFmorph app was developed by the Billinge group at Columbia University.
+The examples in the tutorial part of the manual were made possible by data
+collected by Soham Banerjee at Brookhaven National Laboratory,
+Benjamin Frandsen during his PhD at Columbia Unversity,
+and Long Yang from Tonji University.
+
+The PDFmorph development team is the set of all contributors to the PDFgui project.
```

Comment:
pdfmorph project?

Reply:
Yep, sorry referenced the `PDFgui` blurb